domain/src/main/java/com/example/domain/useCase/weatherData/GetLastTenChosenCitiesUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.domain.useCase.weatherData
+
+import com.example.domain.models.CityModel
+import com.example.domain.repositories.LastTenChosenCitiesRepo
+import io.reactivex.Single
+
+class GetLastTenChosenCitiesUseCase(private val lastTenChosenCitiesRepo: LastTenChosenCitiesRepo){
```

Comment:
It is better not to include the number (Ten) in the name. Later we can decide that we want to show last 15 cities. We will need to rename the class. 
It concerns also other classes and methods. Let's remove the word Ten everywhere.

Reply:
Fixed